src/CompanyService.Data/DepartmentUserRepository.cs

```diff
@@ -55,13 +55,18 @@ public DbDepartmentUser Get(Guid userId, bool includeDepartment)
 
         public IEnumerable<Guid> Find(Guid departmentId, int skipCount, int takeCount, out int totalCount)
         {
+            if (skipCount <= 0 || takeCount <= 0)
+            {
+                throw new BadRequestException("Take count can't be equal or less than 0.");
```

Comment:
message will be incorrect if skipCount <= 0

Reply:
fixed